### DISCUSSION TITLE
https://metrics.lecoq.io/ website get an error 502

### DISCUSSION THREAD

**Discussion by lulunac27a** [lowlighter/metrics#1515] [Q&A]
When I go to https://metrics.lecoq.io/ (any part of the website), I get a 502 Bad Gateway error, so I can't check my metrics

**Comment by lulunac27a** [comment]
Site is now working!

**Reply by lowlighter** [reply]
Hello !

Yes sorry seems the server ran out of storage space at one point and crashed 😅 
I cleaned it and restarted it 👍
